vllm_spyre/v1/worker/spyre_model_runner.py

```diff
@@ -884,8 +892,10 @@ def _prepare_prompt(
         # applies left padding to align with tkv of current decode batch
         # and right padding to align with the next block boundary
         input_tokens, position_ids, mask =\
```

Comment:
Still processing the changes, but wondering if the comment above needs rewording...